pkg/terminal/command.go

```diff
@@ -864,6 +869,17 @@ func parseArgs(args string) ([]string, error) {
 	return v[0], nil
 }
 
+// parseOptionalCount parses an optional count argument.
+func parseOptionalCount(arg string, unspecifiedValue int64) (val int64, err error) {
```

Comment:
unspecifiedValue is always 1, there's no reason to abstract it.

Reply:
OK!